packages/service/lib/schema.js

```diff
@@ -151,6 +151,9 @@ const plugin = {
         }
       },
       additionalProperties: false
+    },
+    options: {
+      type: 'object'
```

Comment:
I assume `additionaProperties` is `true` by default, rigth?

Reply:
Yes it is.